README.adoc

```diff
@@ -1,56 +1,120 @@
-= J-Confs
-[![Build Status](https://travis-ci.org/J-Conf-Team/J-Confs.svg?branch=master)](https://travis-ci.org/J-Conf-Team/J-Confs)
+= J-Confs 
+
+image:https://travis-ci.org/13tomoore/J-Confs.svg?branch=master["Build Status", link="https://travis-ci.org/13tomoore/J-Confs"]
 
 A Java project developed by students during the https://github.com/oliviercailloux/java-course[Java course] taught at Université Paris-Dauphine (in http://www.mido.dauphine.fr/[MIDO] L3 Apprentissage).
 
-The code is hereby published under the MIT License, with their permission.
-
-Delivery schedule for 2019 :
-
-Delivery 1 :
-	Tasks :
-		1) Update dependencies if possible (Junit, Guava, as well as dependencies of different environments)
-		2) Remove the package called "y2018"
-		3) Deleting documents in resources that are not used
-		4) Apply best practices with Guava to the existing system :
-			a) Preconditions (defensive programming)
-			b) StringHelper (for toString)
-			c) getResources
-		5) Perform tests using JUnit for the following classes:
-			a) ConferenceWriter
-			b) Conferencereader or Readfileconference
-			c) Shower Conferences
-			d) ReverseGeoCode
-		6) Delete unused classes
-		7) Create packages to better divide the code according to dependencies between classes:
-			a) Mistletoe
-			b) Conference
-			c) Geo
-		8) Redefine some exception lifts that are too generic
-		9) Match an Icalendar file to a researcher (login.ics)
-		10) Gui with conferneces of a load a calendar:
-			a) Create an independnt Gui
-			b) Create a methode that read all conferences of a file
-			c) unit test for this method
-		11) Resolve "eval mvn" problem in travis
-
-Delivery 2 :
-	Tasks :
-		1)Modify ConferenceGui with possibility:
-			a) The user selects his iCalendar
-			b) All the conferences of this iCalendar are displayed on the screen
-			c) Possibility to edit an Icalendar conference
-			d) Ability to display a map with the location of a selected conference
-			
-Delivery 3 :
-	Tasks :
-		1) Do the JC functionality :
-			a) Method for filling the fields of the canvas with use of the POI library
-			b) Saving the file
-		2) In the continuity of delivery 2 we can also make the History feature:
-			a)When editing a conference from the GUI, the icalendar file is sent to a remote github repository
+The code is hereby published under the *MIT License*, with their permission. +
+
+This link:https://github.com/13tomoore/J-Confs[project] is producted by 5 students : link:https://github.com/aliliKM[Alison KOUDOUSSOU MESSAN], link:https://github.com/jacquesmoati[Jacques MOATI], link:https://github.com/13tomoore[Tom MOORE], link:https://github.com/nikolaspaci[Nikola SPACI] and link:https://github.com/feLlx[Félix VITAS].
```

Comment:
Éviter les `link:` non obligatoires, ça réduit la lisibilité et rend la maintenance plus difficile.
Le lien vers le projet lui-même est inutile ici (on y est déjà !).
Êtes-vous sûr que tous les étudiants veulent avoir leur nom complet inscrit ici ?

Reply:
C'est noté. J'ai demandé à certains membres de mon groupe, cela ne leur posent pas de soucis.

Reply:
Oui mais c’est _tous_ les membres qu’il faut interroger si vous voulez indiquer tous les noms.

Reply:
J'ai pu interroger tous les membres. J'ai eu les accords.